std_msgs/String name
---
moveit_msgs/RobotTrajectory plan
